### PR TITLE
Fix/gwas id bug

### PR DESCRIPTION
**Pull request by alilahrime**
### Bug Fixes
fix typo for id variable in custom dichotomous variable object
addressed feedback on file naming and folder structure

## New features
covariate deletion in case control and quant

### CODE REVIEW

File: src/Analysis/GWASWizard/GWASQuantitative.jsx [hunk 1]
```diff
@@ -115,101 +138,84 @@ const GWASQuantitative = ({ resetGWASType, refreshWorkflows }) => {
...
+              <GWASFormSubmit
+                sourceId={sourceId}
+                numOfPC={numOfPC}
+                mafThreshold={mafThreshold}
+                imputationScore={imputationScore}
+                selectedHare={selectedHare}
+                selectedQuantitativeCohort={selectedCohort}
+                workflowType={'quantitative'}
+                outcome={outcome}
+                // selectedCaseCohort={selectedCaseCohort}
```
Comment (kmhernan) [inline src/Analysis/GWASWizard/GWASQuantitative.jsx:205]: Does this still need to be commented out @alilahrime ?

File: src/Analysis/GWASWizard/GWASQuantitative.jsx [hunk 2]
```diff
@@ -115,101 +138,84 @@ const GWASQuantitative = ({ resetGWASType, refreshWorkflows }) => {
...
+                // selectedControlCohort={selectedControlCohort}
```
Comment (kmhernan) [inline src/Analysis/GWASWizard/GWASQuantitative.jsx:206]: same here?

**Review by kmhernan — APPROVED** [review]
Check my comments, otherwise I think this is ok

File: src/Analysis/GWASWizard/GWASQuantitative.jsx (hunk 1 above)
Comment (alilahrime) [reply src/Analysis/GWASWizard/GWASQuantitative.jsx]: nope, removed comment

File: src/Analysis/GWASWizard/GWASQuantitative.jsx (hunk 2 above)
Comment (alilahrime) [reply src/Analysis/GWASWizard/GWASQuantitative.jsx]: ya

**Review by kmhernan — APPROVED** [review]

**Pull request merged**